blxr-mev.proto

```diff
@@ -30,17 +30,16 @@ message RegisterValidatorResponse{
   string message = 3;
 }
 
-
 // GetHeader
 message GetHeaderRequest{
   uint64 slot = 1;
   string parent_hash = 2;
   string pubkey = 3;
   string client_ip = 4;
   string version = 5;
```

Comment:
proxy do not use getHeader.

Reply:
removed token from the getHeaderRequest. Fixed.